File: src/main/java/com/github/hdzitao/editstarters/initializr/InitializrDependency.java
package com.github.hdzitao.editstarters.initializr;


import com.github.hdzitao.editstarters.dependency.Dependency;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class InitializrDependency extends Dependency {

    private String repository;
    private String bom;
}

File: src/main/kotlin/hdzi/editstarters/springboot/initializr/InitializrDependency.kt
package hdzi.editstarters.springboot.initializr

import hdzi.editstarters.springboot.Dependency

class InitializrDependency(
    override var groupId: String,
    override var artifactId: String,
    var scope: String,
    var version: String?,
    var repository: String?,
    var bom: String?
) : Dependency